app/service.py

```diff
@@ -215,31 +215,31 @@ def verify_answer(answerID, user_data):
     })
 
 def _ensure_user(user_data):
-    user = User.query.filter_by(messenger_id=user_data.messenger_user_id).first()
+    user = User.query.filter_by(messenger_id=user_data['messenger user id']).first()
     if not user:
-        user = User(user_data.messenger_user_id)
+        user = User(user_data['messenger user id'])
         db.session.add(user)
         db.session.commit()
     return user
 
 def _ensure_reading(user_id, article_id):
-    reading = Reading.query.filter_by(user_id=user_id, article_id=article_id)
+    reading = Reading.query.filter_by(user_id=user_id, article_id=article_id).first()
     if not reading:
-        reading = Reading(article_id=article_id, user_id=user_id, attention=0, like=0, refused=False, read=False, score=0)
+        reading = Reading(article_id=article_id, user_id=user_id, attention=0, like=0, refused=0, read=0, score=0)
```

Comment:
Nebylo by lepsi zmigrovat databazi, aby se tam pouzivaly boolovske promenne misto intu na dvojstavove hodnoty?

Reply:
Asi by to bylo čistší, to je pravda. Ale s alembicem jsem ještě nepracoval a bylo by to (minimálně pro mě) mnohem pracnější a nepřineslo by to zase tolik